GCTC-NTGC/gc-digital-talent
# Feature - Update support page style

Resolves #4456.

## Steps to test

1. Compile talentsearch `npm run intl-compile --workspace=talentsearch`
2. Build talentsearch `npm run production --workspace=talentsearch`
3. Navigate to **/support**
4. Verify page looks good and text makes sense
5. Verify page title, subtitle, and breadcrumbs are different in English and French (other parts of this page will be translated as part of #4115)
